### DISCUSSION TITLE
RPM installation issue

### DISCUSSION THREAD

**Discussion by zzcpower** [vectordotdev/vector#13856] [Q&A] [answered]
Hi, there

The error occurred when I tried to install the newest rpm vector-0.23.0-1.x86_64.rpm.
The linux operator system is CentOS-7-x86_64-Minimal-2009.iso

![image](https://user-images.githubusercontent.com/41858357/182983612-ddf16ebd-01c2-4273-9586-23c8cdee39c6.png)

Then I upgrade the libc version to 2.18, but still get the same error
![image](https://user-images.githubusercontent.com/41858357/182983428-b4904a0a-6fb6-4d61-a4c1-60ebb11169ae.png)

Anyone help plz.

**Comment by spencergilbert** [comment, marked as answer]
👋 
Tracked in this issue, https://github.com/vectordotdev/vector/issues/13183, we've had to drop support for centos7 due to changes in `cross-rs`. You can follow that issue for updates.

**Reply by jszwedko** [reply]
Just a note that you can run the musl-based builds on CentOS 7 for now.

**Reply by zzcpower** [reply]
Hi, Thanks for your reply.
Just wondering is there any older version of vector which is available for Centos 7?

**Reply by jszwedko** [reply]
Hi @zzcpower . Yes, you should be able to run an older version of Vector too. I would walk the versions back until you find one that works. Again, you could also run one of the musl-based builds which don't have any libc dependencies.
